int8 nWheelID
---
int8 nWheelDirection
int8 nWheelSpeed
int8 nRetCode
